backend/pkg/api/instances.go

```diff
@@ -631,3 +639,132 @@ func (api *API) instanceStatusHistoryQuery(instanceID, appID, groupID string, li
 		Order(goqu.C("created_ts").Desc()).
 		Limit(uint(limit))
 }
+
+// instanceFactQuery returns a SelectDataset prepared to return all instances
+// that have been checked in during a given duration from a given time.
+func (api *API) instanceFactQuery(t *time.Time, duration *time.Duration) *goqu.SelectDataset {
+	if t == nil {
+		now := time.Now()
+		t = &now
+	}
+
```

Comment:
```suggestion
	if duration == nil {
		d := time.Hour
		duration = &d
	}
```
Removed for debugging but should be added back (maybe comment it out instead?)